libs/ui/separator/brain/src/lib/brn-separator.component.ts

```diff
@@ -1,26 +1,23 @@
-import { Component, Input, booleanAttribute } from '@angular/core';
+import { Component, booleanAttribute, computed, input } from '@angular/core';
 
 export type BrnSeparatorOrientation = 'horizontal' | 'vertical';
+
 @Component({
 	selector: 'brn-separator',
 	standalone: true,
 	template: '',
 	host: {
-		'[role]': '_decorative ? "none" : "separator"',
-		'[attr.aria-orientation]': '_decorative ? undefined : _orientation === "vertical" ? "vertical" : undefined ',
-		'[attr.data-orientation]': '_orientation',
+		'[role]': 'role()',
+		'[attr.aria-orientation]': 'ariaOrientation()',
+		'[attr.data-orientation]': 'dataOrientation()',
 	},
 })
 export class BrnSeparatorComponent {
-	protected _orientation: BrnSeparatorOrientation = 'horizontal';
-	@Input()
-	set orientation(value: BrnSeparatorOrientation) {
-		this._orientation = value;
-	}
+	public readonly dataOrientation = input<BrnSeparatorOrientation>('horizontal', { alias: 'orientation' });
```

Comment:
@ajitzero - Just had a quick question, was there any issue with just keeping this as orientation instead of aliasing? I mention because this seems to break the storybook compilation. Also just wanted to say thanks for you help with the signal migration so far!

Reply:
Since we had `data-` & `aria-` values separately, I decided to rename them to be clear. I didn't run the storybook as there are other issues with the e2e tests, I'll check and update them.

Reply:
@ajitzero thanks so much for picking this up. I think we can stick with orientation as a variable name for dataOrientation and then just derive ariaOrientation for that

Reply:
I've reverted the name change to be just `orientation` again! Also, I've added default values in the Storybook for this API